.github/workflows/welcome-bot.yml

```diff
@@ -0,0 +1,33 @@
+name: "AlloAllo"
```

Comment:
Should we add a comment linking to where this reusable workflow resides and the docs?

Reply:
done in 07d9c9fa1d6b47b3c2ed5ebcfc6473aa15f22c83